### PR TITLE
feat(falco-exporter): added ability to set the grafana folder annotation name

### PR DESCRIPTION
**Pull request by dotdc**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

/kind feature

> If this PR will release a new chart version please make sure to also uncomment the following line:

/kind chart-release

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

/area falco-exporter-chart

**What this PR does / why we need it**:

This PR adds the ability to set the grafana folder annotation name used by the Grafana sidecar.
This is useful when using [folderAnnotation](https://github.com/grafana/helm-charts/blob/main/charts/grafana/values.yaml#L935) in the Grafana Helm Chart.
Set the default value to the previously hardcoded one.

**Which issue(s) this PR fixes**:

N/A

**Special notes for your reviewer**:

**Checklist**

- [X] Chart Version bumped
- [X] Variables are documented in the README.md
- [X] CHANGELOG.md updated

### CODE REVIEW

File: charts/falco-exporter/values.yaml [hunk 1]
```diff
@@ -175,8 +175,10 @@ serviceMonitor:
 grafanaDashboard:
   # -- enabled specifies whether the dashboard should be deployed.
   enabled: false
-  # -- folder where the dashboard is stored by grafana.
+  # -- create and set folder_annotation to specify where the dashboard is stored in grafana.
```
Comment (alacuku) [inline charts/falco-exporter/values.yaml:178]: Comments should start with the variable's name.

File: charts/falco-exporter/values.yaml [hunk 2]
```diff
@@ -175,8 +175,10 @@ serviceMonitor:
...
   folder: ""
+  # -- annotation used by grafana when using folderAnnotation.
+  folder_annotation: "grafana_dashboard_folder"
```
Comment (alacuku) [inline charts/falco-exporter/values.yaml:181]:
```suggestion
  folderAnnotation: "grafana_dashboard_folder"
```

We use camelCase for variable names.

**Review by alacuku — COMMENTED** [review]
Hi @dotdc, thanks for the PR.

I left some suggestions.

File: charts/falco-exporter/values.yaml (hunk 1 above)
Comment (dotdc) [reply charts/falco-exporter/values.yaml]: Should be good now, also applied this to `folderAnnotation` comment

File: charts/falco-exporter/values.yaml (hunk 2 above)
Comment (dotdc) [reply charts/falco-exporter/values.yaml]: done :+1:

**Review by alacuku — APPROVED** [review]
/LGTM

**Pull request merged**